src/main/scala/io/iohk/ethereum/blockchain/data/GenesisDataLoader.scala

```diff
@@ -122,14 +128,12 @@ class GenesisDataLoader(
       case Some(existingGenesisHeader) if existingGenesisHeader.hash == header.hash =>
         log.info("Genesis data already in the database")
         Success(())
-
       case Some(_) =>
         Failure(new RuntimeException("Genesis data present in the database does not match genesis block from file." +
           " Use different directory for running private blockchains."))
-
       case None =>
         // using empty namespace because ephemDataSource.storage already has the namespace-prefixed keys
-        dataSource.update(IndexedSeq(), Nil, ephemDataSource.storage.toSeq)
+        ephemDataSource.storage.toSeq.grouped(dbConfig.batchSize).foreach(toStore => dataSource.update(IndexedSeq(), Nil, toStore))
```

Comment:
As the use of `dbConfig.batchSize` was introduced due to `LevelDB` problems, maybe we should start using this batch size in other updates to the database?